docs/README.md

```diff
@@ -35,6 +35,17 @@ language, grammar, and tone style:
 * If using an abbreviation, spell it out the first time it is used in the document unless it is commonly known. (example: TCP/IP)
 * When referring to a Kubernetes Group (SIG, WG, or UG) do not use the hyphenated form unless it is for a specific purpose such as a file-name or URI.
 
+## reStructured Text Styling Guide
+
+* Use `*` for bullet points.
+* Section headers are created using underlines with text. Underlining hierarchy is:
+    * `*` for chapters
+    * `=` for sections
+    * `-` for subsections
+    * `^` for subsubsections
+    * `"` for paragraphs
+* In code blocks, a required argument is represented by `<required>` and optional arguments are represented as `[optional]`.
```

Comment:
```suggestion
```
This can be skipped. It's just a convention followed while documenting command-line syntax and is not applicable to all code blocks.
For example Microsoft Windows commands follow this convention - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/command-line-syntax-key

Reply:
Instead, should I specify that this is for command line syntax then?

Reply:
I think we can skip this part for now. Although it's a convention, the actual usage of the command documented may differ.